app/controllers/application_controller.rb

```diff
@@ -1,3 +1,4 @@
 class ApplicationController < ActionController::Base
   protect_from_forgery with: :exception
+  include ApplicationHelper
```

Comment:
In recent versions of Rails all the helper modules are loaded automatically. So generally, no need to explicitly include them.

Reply:
Good to know, thanks Elle!